src/sections/Why/History.tsx

```diff
@@ -10,16 +10,13 @@ export const History = () => (
       <Text as="span" variant="emphasis">
         History:
       </Text>{' '}
-      Software engineer{' '}
-      <Link href="https://twitter.com/tmeasday">Tom Coleman</Link> introduced
+      Software engineer <Link href="https://twitter.com/tmeasday">Tom Coleman</Link> introduced
       Component Driven in{' '}
-      <Link href="https://blog.hichroma.com/component-driven-development-ce1109d56c8e">
-        2017
-      </Link>{' '}
-      to describe the shift in UI development toward component architectures.
-      The idea of modular software has many parallels in movements such as
-      microservices and containerization. Historical precedents also include
-      lean manufacturing and mass manufacturing circa early 20th century.
+      <Link href="https://blog.hichroma.com/component-driven-development-ce1109d56c8e">2017</Link>{' '}
+      to describe the shift in UI development toward component architectures and processes. The idea
```

Comment:
I wanted to add "processes" here because the shift to component-based UIs in the web started well before 2017 (even if React invented it[1], React was released in 2013)

[1] Of course it didn't.